plugins/modules/vmware_guest.py

```diff
@@ -1563,9 +1565,11 @@ def configure_network(self, vm_obj):
             else:
                 # Default device type is vmxnet3, VMware best practice
                 device_type = network_devices[key].get('device_type', 'vmxnet3')
+                add_key = add_key + 1
                 nic = self.device_helper.create_nic(device_type,
-                                                    'Network Adapter %s' % (key + 1),
-                                                    network_devices[key])
+                                                    'Network Adapter %s' % (add_key),
+                                                    network_devices[key],
+                                                    add_key)
```

Comment:
I think it is ok even when single-digit integer, but the API document seems to use negative integers as a temporary key.

> (for example, by using unique negative integers as temporary keys).

https://vdc-download.vmware.com/vmwb-repository/dcr-public/b50dcbbf-051d-4204-a3e7-e1b618c1e384/538cf2ec-b34f-4bae-a332-3820ef9e7773/vim.vm.device.VirtualDevice.html

I checked on Developer Center of vCenter used negative integers indeed to temp key. (For example, `-100` `-101` and so on)
What do you think?